5_databases/week1/homework.sql

```diff
@@ -0,0 +1,60 @@
+-- Active: 1699628373418@@127.0.0.1@3306@hyf_lesson1
+
+-- 1.Find out how many tasks are in the task table
+SELECT count (*)
```

Comment:
`count (*)` - space between count and what it is counting `(*)` is not a valid syntax and will return an error. Use `count(*)`